modules/appeals_api/app/services/appeals_api/events/appeal_received.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module AppealsApi
+  module Events
+    class AppealReceived
+      def initialize(opts)
+        @opts = opts
+        raise InvalidKeys unless required_keys?
+      end
+
+      def hlr_received
+        log_error unless email
+
+        template_type = 'higher_level_review_received'
+        template_id = template_id(template_type)
+
+        vanotify_service.send_email(
+          email_address: email,
+          template_id: template_id,
+          personalisation: {
+            'veteran_first_name' => opts['veteran_first_name'],
+            'veteran_last_name' => opts['veteran_last_name'],
+            'date_submitted' => opts['date_submitted']
+          }
+        )
+      end
+
+      private
+
+      attr_accessor :opts
+
+      def vanotify_service
+        @vanotify_service ||= VaNotify::Service.new(Settings.vanotify.services.lighthouse.api_key)
+      end
+
+      def template_id(template)
+        Settings.vanotify.services.va_gov.template_id.public_send(template)
+      end
+
+      def log_error
+        Rails.logger.error 'No email present for AppealsApi::AppealReceived notification'
```

Comment:
What do you think about passing in the HLR's guid into the `opts` so that we can know which record got this far? It's almost certain not to happen with our other checks, but if I saw this error in the logs I wouldn't really know where to start looking.

Reply:
Maybe also add that this is an error from the HLR-based event handler, so we know to look for HLRs (assuming we'll expand this to NODs or SC's later)

Reply:
I can finagle it, but technically speaking, this file is not specific to HLR, it's for any appeal. The callback is specific to hlr, being `hlr_received`.  I'll figure it out! I like the idea.